include/admin.php

```diff
@@ -133,7 +133,7 @@ public static function network_page_admin_duplicate_site() {
             // Load template if at least one Site is available
             if( $site_list ) {
 
-                $select_site_list = MUCD_Admin::select_site_list($site_list, $data['source']);
+                $select_site_list = apply_filters( 'mucd_select_site_list', MUCD_Admin::select_site_list( $site_list, $data['source'] ) );

```

Comment:
In this case, the `MUCD_Admin::select_site_list` query will still run. We need a way to bypass that. Might look at something like this to instead: https://github.com/WebDevStudios/CMB2/blob/master/includes/CMB2_Field.php#L280